packages/solid/components/Column/Column.tsx

```diff
@@ -35,10 +35,10 @@ const Column: Component<ColumnProps> = (props: ColumnProps) => {
       onDown={chainFunctions<KeyHandler | undefined>(props.onDown, onDown)}
       selected={props.selected || 0}
       forwardFocus={onGridFocus}
-      onBeforeLayout={
+      onLayout={
```

Comment:
is there a difference between these two, or was the lifecycle method just renamed?

Reply:
I'd like to get rid of onBeforeLayout - I don't think there is much use case for it.